Http.Mqtt.Connector.Svc/MqttDataSink.cs

```diff
@@ -75,7 +75,11 @@ public async Task PushDataAsync(JsonDocument data, CancellationToken stoppingTok
 
         var mqtt_application_message = new MqttApplicationMessage(_topic, MqttQualityOfServiceLevel.AtLeastOnce)
         {
-            PayloadSegment = new ArraySegment<byte>(Encoding.UTF8.GetBytes(data.RootElement.GetRawText()))
+            PayloadSegment = new ArraySegment<byte>(Encoding.UTF8.GetBytes(data.RootElement.GetRawText())),
+            UserProperties = new List<MqttUserProperty>
+            {
+                new MqttUserProperty("OriginalUrl", _originalUrl)
```

Comment:
Where's _originalUrl defined?

Reply:
Also, consider changing property to original source instead, as we may have a different data source in future.